pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -653,7 +656,13 @@ stdenv.mkDerivation ({
       mkdir -p "$packageConfDir"
       ${setupCommand} register --gen-pkg-config=$packageConfFile
       if [ -d "$packageConfFile" ]; then
-        mv "$packageConfFile/"* "$packageConfDir"
+        ${if (buildTarget == "")
+        then ''mv "$packageConfFile/"* "$packageConfDir"''
+        else ''
+          for file in "$packageConfFile/"*; do
+            mv "$file" "$packageConfDir"
+          done
+        ''}
```

Comment:
This ugly if/else is here to prevent rebuilding too many packages. It's from the 4th commit: "haskell-generic-build: try to prevent mass rebuild". It should probably not be merged?